src/libs/suggest.js

```diff
@@ -79,7 +79,7 @@ export const modifyList = (items, withGeoloc) => {
     items.splice(0, 0, NavigatorGeolocalisationPoi.getInstance());
   }
 
-  if (items.length === 0 || items.length === 1 && withGeoloc) {
+  if (query.length > 0 && items.length === 0 || items.length === 1 && withGeoloc) {
```

Comment:
Similarly, the error should not be returned on empty input with geoloc:
```suggestion
  if (query.length > 0 && (items.length === 0 || items.length === 1 && withGeoloc)) {
```